pkg/lib/cockpit-components-password.scss

```
@@ -1,7 +1,12 @@
+@import "global-variables";
+@import "@patternfly/patternfly/utilities/Text/text.scss";
+
 .ct-password-strength-meter {
   grid-gap: var(--pf-v5-global--spacer--xs);
 
   .pf-v5-c-progress__description, .pf-v5-c-progress__status {
     display: none;
   }
+
+  inline-size: var(--pf-v5-global--spacer--2xl);
 }
```

```suggestion
.ct-password-strength-meter {
  grid-gap: var(--pf-v5-global--spacer--xs);
  inline-size: var(--pf-v5-global--spacer--2xl);

  .pf-v5-c-progress__description, .pf-v5-c-progress__status {
    display: none;
  }
}
```

Parent CSS should be grouped together at the top. (This doesn't change anything, but it's better when it's not scattered.)